### PR TITLE
Swift 4_2 Update

### PR DESCRIPTION
**Pull request by justingluck93**
This PR is **ready** for review.

### Risk
No API changes

### Testing Plan
Pull branch and run the swift example app

### Summary
Changed a few lines of code to reflect swift 4.2 syntax

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: SmartDeviceLink-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3815,7 +3815,7 @@
 				5D4019AF1A76EC350006B0C2 /* SDL Example.app */,
 				5D61FA1C1A84237100846EE7 /* SmartDeviceLink.framework */,
 				5D61FA261A84237100846EE7 /* SmartDeviceLinkTests.xctest */,
-				88295697207CF68800EF056C /* SDL Example.app */,
+				88295697207CF68800EF056C /* SDL Example Swift.app */,
```
Comment (joeljfischer) [inline SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: We don't want to update the swift example app name, it should be identical to the obj-c example app name: `SDL Example`.

File: SmartDeviceLink-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -6573,7 +6573,7 @@
 			);
 			name = "SmartDeviceLink-Example-Swift";
 			productName = "SmartDeviceLink-iOS";
-			productReference = 88295697207CF68800EF056C /* SDL Example.app */;
+			productReference = 88295697207CF68800EF056C /* SDL Example Swift.app */;
```
Comment (joeljfischer) [inline SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: See above

File: SmartDeviceLink-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (justingluck93) [reply SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: fixed

File: SmartDeviceLink-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (justingluck93) [reply SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: fixed

File: SmartDeviceLink-iOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2847,7 +2847,7 @@
 		8818ADD62100FC18007D6F19 /* SDLTurnSignal.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = SDLTurnSignal.h; sourceTree = "<group>"; };
 		8818ADD72100FC18007D6F19 /* SDLTurnSignal.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = SDLTurnSignal.m; sourceTree = "<group>"; };
 		8818ADDC2100FE0C007D6F19 /* SDLTurnSignalSpec.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = SDLTurnSignalSpec.m; sourceTree = "<group>"; };
-		88295697207CF68800EF056C /* SDL Example.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; name = "SDL Example.app"; path = "SDL Example Swift.app"; sourceTree = BUILT_PRODUCTS_DIR; };
+		88295697207CF68800EF056C /* SDL Example.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "SDL Example.app"; sourceTree = BUILT_PRODUCTS_DIR; };
```
Comment (joeljfischer) [inline SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: This is probably the cause of the example swift app crashing on launch

Comment (justingluck93) [reply SmartDeviceLink-iOS.xcodeproj/project.pbxproj]: what is wrong with it?

**Pull request merged**